modules/olmv1-installing-an-operator.adoc

```diff
@@ -142,48 +160,95 @@ clusterextension.olm.operatorframework.io/pipelines-operator created
 $ oc get clusterextension pipelines-operator -o yaml
 ----
 +
---
-include::snippets/olmv1-version-range-cr-adomn.adoc[]
---
-+
 .Example output
 [%collapsible]
 ====
 [source,text]
 ----
-apiVersion: olm.operatorframework.io/v1alpha1
-kind: ClusterExtension
+apiVersion: v1
+items:
+- apiVersion: olm.operatorframework.io/v1alpha1
+  kind: ClusterExtension
+  metadata:
+    annotations:
+      kubectl.kubernetes.io/last-applied-configuration: |
+        {"apiVersion":"olm.operatorframework.io/v1alpha1","kind":"ClusterExtension","metadata":{"annotations":{},"name":"pipelines-operator"},"spec":{"channel":"latest","installNamespace":"openshift-marketplace","packageName":"openshift-pipelines-operator-rh","pollInterval":"30m"}}
```

Comment:
@michaelryanpeter it is not changed here `"installNamespace":"openshift-marketplace"`.